### ISSUE TITLE
BUG-039: Multiple WCAG 2.1 AA Accessibility Violations Detected (Critical + 34 Issues)

### ISSUE THREAD

**Issue by StevenOyar** [StevenOyar/wk-6-StevenOyar-1#83]
---

**Related TC:** TC-096
**Related Requirement:** FR-071

---

**Severity:** Critical
**Priority:** High
**Status:** Open

**Environment:**

* Browser: Chrome  with **axe DevTools** extension installed 
* OS: Windows 11
* Device: Desktop
* Resolution: 1920×1080
* URL: http://localhost:3000/

---

**Steps to Reproduce:**

1. Open the application in Chrome.
2. Install and enable **axe DevTools** accessibility extension.
3. Navigate to each application page (Home, Login, Register, Dashboard, Pickup Request, Admin Panel).
4. Run an **axe accessibility scan** on all pages.
5. Review the list of violations generated by the tool

---

**Expected Result:**

* No **Critical** WCAG 2.1 AA violations.
* Minimal or no accessibility violations across the application.
* Application should meet **WCAG 2.1 AA** compliance requirements per FR-071.

**Actual Result:**

* 1 critical violation** detected.
* 34 additional WCAG 2.1 AA violations** found across multiple pages.
* Issues include "Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds"
*

----

**Evidence**
https://axe.deque.com/axe-devtools/user-flows/953ac9e9-6da2-4f41-b86a-468fcc9b3dbc 

<img width="1912" height="867" alt="Image" src="https://github.com/user-attachments/assets/cb11b7ee-f57e-4429-a500-2bd09d460842" />

----

**Reported By:** @StevenOyar 
**Date Reported:** 14/11/2025
